RELEASE_NOTES

```diff
@@ -1,6 +1,33 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.7.0 (13 March 2025)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Improvements:
+* implement exportvariable filter (#288, #291)
+* bot only reports moving to trash_bin when relevant (#292)
+* enabling two ways to submit/receive jobs (#297)
```

Comment:
Maybe this should be worded differently?

It adds an alternative way (doesn't introduce two ways).
Should also briefly mention how this alternative way differs, what the relevant config settings are?